fed/_private/constants.py

```diff
@@ -17,6 +17,8 @@
 
 KEY_OF_JOB_CONFIG = b"JOB_CONFIG"
 
+KEY_OF_METADATA = b"METADATA"
```

Comment:
b"METADATA": is this too general to be distinguished?

Reply:
INDEED